.readthedocs.yml

```diff
@@ -13,9 +13,9 @@ python:
   version: 3.7
   install:
     # Ensure the Find External Link PyTorch version is installed
-    - requirements: requirements_dev.txt
+    - requirements: requirements.txt
```

Comment:
Good catch. 

> # Ensure the Find External Link PyTorch version is installed

is out of date now (it referred to `--find-links`). And actually this entire step can probably be dropped now, since it should be covered by the `pip` step.

Reply:
Awesome!